lib/Widgets/Sidebar/Sidebar.vala

```diff
@@ -0,0 +1,43 @@
+
+namespace Granite.Widgets {
+    public class Sidebar : Gtk.ListBox {
+    
+        public Sidebar () {
+            Object ();
+        }
+        
+        construct {
+            build_ui ();
```

Comment:
Don't do this. This is silly. A method just to call another method doesn't make sense.